src/bc/baseConfig.groovy

```diff
@@ -0,0 +1,22 @@
+    
+package bc;
+
+class baseConfig {
+  // Wait timeout in minutes
+  public static final int WAIT_TIMEOUT = 10
+
+  // Deployment Environment TAGs
+  public static final String[] DEPLOYMENT_ENVIRONMENT_TAGS = ['dev', 'test', 'prod']
+
+  // The name of the project namespace(s).
+  public static final String  NAME_SPACE = 'bztwou'
```

Comment:
Please update to use correct namespace.  Do a full search.  This is not the only place that needs to be updated.